lib/sortability/active_record/base.rb

```diff
@@ -56,9 +56,12 @@ def sortable_methods(options = {})
                 send(setter_mname, max_val + 1)
               elsif peers.to_a.any? { |p| p != self && p.send(on) == val }
                 # Make a gap for the record
-                peers.where{__send__(on) >= val}.reorder(nil)
+                at = self.class.arel_table
```

Comment:
Ah, good to know: I'm about to do this same thing inside publishable, I think. :-)